CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 0.0.14
+* Always use the desired knife config
+* Robocop compliance
+* Add "partial failure" reporting
+* Fix race conditions
+* Better detection for existing chef-zero instances
+* README corrections
+* noop transport fixes
```

Comment:
s/noop/NoOp/

Reply:
The *class* is NoOp, but the argument to `--transport` is `noop`.